fivetran_provider_async/utils/operator_utils.py

```diff
@@ -0,0 +1,144 @@
+from typing import List
+
+from openlineage.client.facet import (
+    DataSourceDatasetFacet,
+    SchemaDatasetFacet,
+    SchemaField,
+)
+from openlineage.client.run import Dataset
+
+
+def _get_table_id(table_src, table, tables, loc) -> str:
+    """
```

Comment:
The arguments for this method seemed a little confusing to me - `table_src`, `table`, `tables`

Can `table_src` derive from `table`?

Reply:
I can rename these. `table_src` cannot derive from `table` as it is the key in the schemas API response dictionary, and `table` is the value. But I can just pass that entire object together as `schema_api_table` and then have `tables_api_tables` or something like that, to explicitly show where these table dicts are coming from.

Reply:
lmk if this changed is good!